setup/docker/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+which mvn &> /dev/null || { echo >&2 "Maven package cant be found on path. Aborting."; exit 1; }
+which docker &> /dev/null  || { echo >&2 "Docker package cant be found on path. Aborting."; exit 1; }
+mvn package || { echo >&2 "Maven package has failed. Aborting."; exit 1; }
+
+export company="tananaev"
+export software="traccar"
+export version=$(head -n 10 ./pom.xml |grep version|cut -d ">" -f2|cut -d"<" -f1)
+
+tmp="./setup/docker/tmp"
+
+mkdir -p ${tmp}/data/database
```

Comment:
Since we are using ./setup/unix/traccar.xml, it points to:
<entry key='database.url'>jdbc:h2:/opt/traccar/data/database</entry>

and since we copy ./tmp/data to inside of container, we need to create the db folder before copy all data.